letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -342,6 +342,21 @@ def _choose_vhost_from_list(self, target_name, temp=False):
         self.assoc[target_name] = vhost
         return vhost
 
+    def included_in_wildcard(self, names, target_name):
+        """Helper function to see if alias is covered by wildcard"""
```

Comment:
When looking into this, I found that `ServerAlias` directives can also include a protocol/port. For example, `https://*example.com:443` is a valid server alias. While this code won't blow up if it encounters one of these directives, they aren't correctly parsed as a `ServerAlias`.